sechub-shared-kernel/src/main/java/com/daimler/sechub/sharedkernel/configuration/login/Page.java

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sharedkernel.configuration.login;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Page {
+    Optional<List<Action>> actions = Optional.empty();
```

Comment:
Please set this to private, so `getActions` is the defined way to access this.

Reply:
For testing purposes is cannot be `private`.